### ISSUE TITLE
Offering Copywritting Contribution To Your Project 

### ISSUE THREAD

**Issue by Kelita** [CityZenApp/Android-Development#101]
Hello, I'm Kelita

I would love to contribute to your project. I went through your repository and discovered some documentation are missing which I would like to contribute to. 

I am waiting your response and would be honored if giving the privilege to do so.

**Comment by rskikuli** [comment]
Thank you for your interest @Kelita . One of the areas you can contribute is localization, this means that you can translate CityZen to your language. Here is how you can do that: https://hosted.weblate.org/projects/cityzen/strings/. Also a good place to start getting to know the community better is by joining our IRC channel https://twitter.com/CityZenappco/status/1006657240546324482.
If there are any other areas you think you can get involved feel free to propose. 

ps: can you share how found us?

**Comment by Kelita** [comment]
Thanks for your reply. However, I'm more of a writer and content creator than a translator.

The documentation contributions I originally planned to do was 
Create a FAQs document which is a compilation of basic and advanced questions users may likely ask when using the app.

Then a Privacy Policy document which is really important for any project because it gives users information about what personal information they part with when they use the app, how the information is collected and utilized and also how secured is the information in your hands.

Then about us which will be integrated in the document will give brief information about the project, the dev team

**Comment by rskikuli** [comment]
Than sounds great. Highly appreciated. Should we start with the FAQs? I opened a pad where you can start with a draft and more people can jump in and improve it even more: https://pad.openlabs.cc/p/CityZen_app_FAQs

What do you think?

**Comment by Kelita** [comment]
Okay, Let's start with the FAQs, I will write them then send to you. You will check for correction or amendment to the FAQs written. You can also drop any details you want to include. 

I forgot to answer how I found you. I discovered you through Utopian discord channel.

**Comment by rskikuli** [comment]
Cool. Really happy to have you join our little but ambitious open source project :) Looking forward.

**Comment by Kelita** [comment]
I am honored. I will be having some questions which would be needing your answers, how do I contact you? Or should I keep dropping them here for you?

First question
- How does Cityzen find your point of interest?

**Comment by rskikuli** [comment]
Hey @Kelita, I'm presenting CityZen these days at the [State of the Map](https://stateofthemap.org/) conference in Milan and will have limited access to my computer. I'll definitely check this next week. Thanks for your understanding.

**Comment by Kelita** [comment]
Alright! I will definitely hold on till when you are available. No rush. Thanks.

**Comment by rskikuli** [comment]
I added some more content and published the page on the wiki https://github.com/CityZenApp/Android-Development/wiki/Frequently-Asked-Questions. Please have a look. 
You mentioned that the app contains adds. Where did you see this? Please keep in mind that CityZen does not have any ads! 
Also, can you please send me the link to the Utopian forum where CityZen is mentioned? Thanks again.

**Comment by Kelita** [comment]
Thanks for the time. The Ads! was a mistake. Thanks for also adding it to the wiki section of the project. I have also update the pad. I can I proceed to the About Us or Privacy Policy?

Here is the link to the discord channel https://discord.gg/Vn2Ntpe

**Comment by rskikuli** [comment]
Yes, I think the FAQs looks good at this stage. We can update and improve it later. I think the 'privacy policy' session of the app would be the next step.

**Comment by rskikuli** [comment]
More copywritig is needed for the new 'support us' page we are working on. Here is the issue: https://github.com/CityZenApp/CityZenApp.github.io/issues/10. Let me know if you have any questions. Thanks.

**Comment by Kelita** [comment]
Thanks a lot. I will move to the Privacy Policy. You can add me as a contributor. Use Kelita as the name.  No problem about the crypto payment. The project is open source

**Comment by Kelita** [comment]
I see you have a Privacy policy already, I will move to the About Us or do you want to modify it?

**Comment by Kelita** [comment]
I went through your app also and discovered you have an about us. I think my work is done then. It was nice working with you.
